docs/static_landing_page/index.html

```diff
@@ -27,30 +27,30 @@
       		<div class="collapse navbar-collapse" id="navbarResponsive">
 		        <ul class="navbar-nav pt-2 mt-1 mr-xl-auto ml-xl-2 pl-xl-1 text-center text-nowrap">
 				  <li class="nav-item px-1">
-					  <a class="nav-link" href=./v0.6.2/quickstart.html>Getting Started</a>
+					  <a class="nav-link" href=./v0.7.0/quickstart.html>Getting Started</a>
```

Comment:
would it be possible to make this a variable and change it once in this file?

Reply:
Yes! Done!